project/AppDependencies.scala

```diff
@@ -32,6 +32,9 @@ object AppDependencies {
     "uk.gov.hmrc"            %% "bootstrap-test-play-27"   % "2.25.0"         % scope,
     "uk.gov.hmrc"            %% "domain"                   % "5.11.0-play-27" % scope,
     "com.vladsch.flexmark"    % "flexmark-all"             % "0.35.10"        % scope,
+    "uk.gov.hmrc"            %% "webdriver-factory"        % "0.18.0"         % scope,
+    "org.scalatestplus"      %% "selenium-3-141"           % "3.2.0.0"        % scope,
+    "com.typesafe"            % "config"                   % "1.3.2"          % scope,
```

Comment:
Shouldn't need this `config` dependency I think, it should be bundled in with Play.

Reply:
Correct, removed (not pushed yet, I'll batch a few commits up)